samples/clip-geometry/README.metadata.json

```diff
@@ -6,28 +6,13 @@
     "images": [
         "clip-geometry.png"
     ],
-    "keywords": [
-        "analysis",
-        "clip",
-        "geometry",
-        "Envelope",
-        "Geometry",
-        "GeometryEngine.clip",
-        "Graphic",
-        "GraphicsOverlay"
-    ],
-    "language": "java",
-    "redirect_from": [
-        "/android/latest/sample-code/clip-geometry.htm"
-    ],
-    "relevant_apis": [
-        "Envelope",
-        "Geometry",
-        "GeometryEngine.clip",
-        "Graphic",
-        "GraphicsOverlay"
-    ],
+    "keywords": [ ],
+    "language": "kotlin",
+    "redirect_from": "",
+    "relevant_apis": [ ],
```

Comment:
@darryl-lynch you'll want to run the metadata_updater.py which will pull in keywords and relevant_apis from the README

https://devtopia.esri.com/runtime/common-samples/tree/main/tools/Android

Reply:
Turns out I ran the metadata_checker before not the metadata_updater, whoops